lang/ru.po

```diff
@@ -0,0 +1,95 @@
+msgid ""
+msgstr ""
+"Language: ru\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n==1 ? 0 : 1);\n"
+
+# A place for you to credit yourself!
+# Put something like "Latin translation by Jvlivs Gaivs\n" here.
+msgid "TRANSLATION_AUTHOR\n"
+msgstr "stacey-impossible\n"
+
+msgid "FREE_PACKAGES_COUNT\n"
+msgid_plural "FREE_PACKAGES_COUNT\n"
+msgstr[0] "%d свободных пакетов\n"
+msgstr[1] "%d свободных пакетов\n"
+
+msgid "NONFREE_PACKAGES_COUNT\n"
+msgid_plural "NONFREE_PACKAGES_COUNT\n"
+msgstr[0] "%d проприетарных пакетов\n"
+msgstr[1] "%d проприетарных пакетов\n"
+
+msgid "RMS_HAPPY\n"
+msgstr "Только свободные пакеты - Ричард Столлман гордился бы!\n"
+
+msgid "RMS_DISAPPOINTED\n"
+msgstr "Больше 10%% проприетарных пакетов. Вы не цените свободу?\n"
+
+msgid "HELP_USAGE\n"
+msgstr "Команда: vrms-rpm [флаг]\n"
+
+msgid "HELP_OPTION_ASCII\n"
+msgstr "    Показывать ASCII-арт с Ричардом Столлманом, если не нашлось\n"
+       "    проприетарных пакетов или если их меньше 10%%\n"
+
+msgid "HELP_OPTION_COLOUR\n"
+msgstr "    Настроить цветной вывод терминала. Значение по умолчанию: 'auto',\n"
+       "    которое использует цветной вывод в терминал, но не в файл или pipe.\n"
+
+msgid "HELP_OPTION_DESCRIBE\n"
+msgstr "    Отображать короткие описания пакетов при выводе их списка.\n"
+
+msgid "HELP_OPTION_EXPLAIN\n"
+msgstr "    Отображать лицензии пакетов при выводе их списка,\n"
+       "    чтобы убедиться в правильности класиффикации свободных/проприетарных пакетов.\n"
```

Comment:
The lines for this message should not exceed 80 characters (just the text in quotes, not the whole line). You can probably move a word or two to the line above.